config.py

```diff
@@ -10,6 +10,7 @@ class Config(object):
 class DevConfig(Config):
     DEBUG = True
     DATABASE_URI = getenv('DEV_DATABASE_URI', '')
```

Comment:
We are using:
SQLALCHEMY_DATABASE_URI

so we can remove DATABASE_URI property